shared/teams/render.js

```diff
@@ -42,7 +42,7 @@ type BetaNoteProps = {
   onReadDoc: () => void,
 }
 
-const BetaNote = (props: BetaNoteProps) => (
+export const BetaNote = (props: BetaNoteProps) => (
```

Comment:
we usually do an export {Header, BetaNote} at the bottom vs inlining the exports

Reply:
Done